common/lib/opentelemetry/common/class_scoped_attributes.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Common
+    # Attributecontains common helpers for atribute propagation
+    # Various classes should extend this module to get `with_attributes`
+    # functionality scoped to that class.
+    module ClassScopedAttributes
+      extend self
+
+      def current_attributes_hash
```

Comment:
This should be `private`. We should not expose the context key outside of this module.

Reply:
28c1499689fef32d8ee98abbf44b5faef4f9068e

Reply:
> This should be `private`. We should not expose the context key outside of this module.

bf98ea86e1817c5f4095aab4310219da5c0a648d